docs/en/latest/plugins/proxy-cache.md

```diff
@@ -25,64 +26,66 @@ description: This document contains information about the Apache APISIX proxy-ca
+| cache_strategy     | string         | False    | disk                      | ["disk","memory"]       | Caching strategy. Cache on disk or in memory.          |
+| cache_zone         | string         | False    | disk_cache_one            |                         | Cache zone used with the caching strategy. The value should match one of the cache zones defined in the [configuration files](#static-configurations) and should correspond to the caching strategy. For example, when using the in-memory caching strategy, you should use an in-memory cache zone. |
+| cache_key          | array[string]  | False    | ["$host", "$request_uri"] |                         | Key to use for caching. Support [Nginx variables](https://nginx.org/en/docs/varindex.html) and constant strings in values. Variables should be prefixed with a `$` sign.    |
+| cache_bypass       | array[string]  | False    |                           |                         | One or more parameters to parse value from, such that if any of the values is not empty and is not equal to `0`, response will not be retrieved from cache. Support [Nginx variables](https://nginx.org/en/docs/varindex.html) and constant strings in values. Variables should be prefixed with a `$` sign.     |
+| cache_method       | array[string]  | False    | ["GET", "HEAD"]           | ["GET", "POST", "HEAD"] | Request methods of which the response should be cached.       |
+| cache_http_status  | array[integer] | False    | [200, 301, 404]           | [200, 599]              | Response HTTP status codes of which the response should be cached.   |
+| hide_cache_headers | boolean        | False    | false                     |                         | If true, hide `Expires` and `Cache-Control` response headers.   |
+| cache_control      | boolean        | False    | false                     |                         | If true, comply with `Cache-Control` behavior in the HTTP specification. Only valid for in-memory strategy.     |
+| no_cache           | array[string]  | False    |                           |                         | One or more parameters to parse value from, such that if any of the values is not empty and is not equal to `0`, response will not be cached. Support [Nginx variables](https://nginx.org/en/docs/varindex.html) and constant strings in values. Variables should be prefixed with a `$` sign.       |
+| cache_ttl          | integer        | False    | 300          |        >=1          | Cache time to live (TTL) in seconds when caching in memory. To adjust the TTL when caching on disk, update `cache_ttl` in the [configuration files](#static-configurations). The TTL value is evaluated in conjunction with the values in the response headers  [`Cache-Control`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control) and [`Expires`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Expires) received from the Upstream service.     |
```

Comment:
Shall we capitalize "s" in "service(s)"?

Reply:
> from the Upstream service

Here? No need as it is not referring to the APISIX Service resource